PWGGA/GammaConv/AliCaloPhotonCuts.cxx

```diff
@@ -4987,6 +4987,8 @@ void AliCaloPhotonCuts::ApplyNonLinearity(AliVCluster* cluster, Int_t isMC)
             energy /= FunctionNL_kSDM(energy, 0.962095*0.9991*0.9993, -3.63967, -0.747825) ;
             energy /= FunctionNL_kSDM(energy, 0.988922, -4.47811, -0.132757);
             energy /= FunctionNL_kSDM(energy, 0.99738, -4.82724, -0.281305);
+          } else if(fClusterType==2){ //const fit
+            energy *= 1.0111857903;
```

Comment:
![meanmassratio_lhc10bcdef-pythia6-phoss](https://user-images.githubusercontent.com/30235576/38421243-c6876bf4-39a6-11e8-8004-55ca28793e47.png)
NonLin looks constant in first iteration at low energies